src/bfrini.f90.in

```diff
@@ -47,7 +47,7 @@ COMMON /TABLEF/ CDMF
 COMMON /TNKRCP/ ITRYR,ITRMO,ITRDY,ITRHR,ITRMI,CTRT
 COMMON /ENDORD/ IBLOCK,IORDBE(4),IORDLE(4)
 
-character*100 CMTDIR
+character*240 CMTDIR
```

Comment:
Where does the 240 come from?

Reply:
No place in particular, but 100 seemed like it might be a bit low for some pathnames that could conceivably be set using MASTER_TABLE_DIR on some systems, so I went ahead and upped it here and in mtinfo.